tf2_eigen/CMakeLists.txt

```diff
@@ -35,20 +44,14 @@ if(BUILD_TESTING)
 
   ament_add_gtest(tf2_eigen-test test/tf2_eigen-test.cpp)
   if(TARGET tf2_eigen-test)
-    target_include_directories(tf2_eigen-test PUBLIC
-      include)
-    ament_target_dependencies(tf2_eigen-test
-      "tf2"
-      "tf2_ros")
-    target_include_directories(tf2_eigen-test PUBLIC
-      ${Eigen3_INCLUDE_DIRS})
+    target_link_libraries(tf2_eigen-test tf2_eigen)
```

Comment:
It looks like this test depends on geometry_msgs, tf2, tf2_eigen, and Eigen.  Shouldn't we be listing all of them in this `target_link_libraries` call?

Reply:
In https://github.com/ros2/geometry2/pull/493/commits/7dac8dd51997b0d0892a0e527e12c5f738b98d24 , https://github.com/ros2/geometry2/pull/493/commits/2a9df115fcde75dbd54ddc9347e55d66c1dbedd7 , and https://github.com/ros2/geometry2/pull/493/commits/e78a34d4724545ce557b9cf568d66819dfcd6c6d I double checked the dependencies, and applied the method we came up with in #496 of documenting some dependencies of tests instead of linking them.